omnibus/config/software/datadog-agent.rb

```diff
@@ -78,6 +78,7 @@
     conf_dir = "#{install_dir}/etc/datadog-agent"
   end
   mkdir conf_dir
+  mkdir "#{conf_dir}/runtime-security.d"
```

Comment:
could make sense to only include on Linux, since it's not available on other platforms

Reply:
+1

Reply:
Indeed. Fixed